openmetadata-service/src/main/java/org/openmetadata/service/security/DefaultAuthorizer.java

```diff
@@ -97,6 +98,13 @@ public boolean shouldMaskPasswords(SecurityContext securityContext) {
     return !subjectContext.isBot();
   }
 
+  /** In 1.2, evaluate policies here instead of just checking the subject */
+  @Override
+  public boolean authorizePII(SecurityContext securityContext, EntityReference owner) {
+    SubjectContext subjectContext = getSubjectContext(securityContext);
+    return subjectContext.isAdmin() || subjectContext.isBot() || subjectContext.isOwner(owner);
```

Comment:
are we only allowing admin /bots /owner to view the Sample Data?

Reply:
Everyone can view Sample Data unless there are columns flagged as PII.Sensitive. In that case, only admins/bots or the table owner(s) can view those specific columns.

On 1.2, we'll improve this by adding policy management